images/base/Dockerfile

```diff
@@ -6,8 +6,16 @@
 #
 FROM centos:centos7
 
-RUN INSTALL_PKGS="which git tar wget hostname sysvinit-tools util-linux bsdtar epel-release \
-      socat ethtool device-mapper iptables tree findutils nmap-ncat e2fsprogs xfsprogs lsof device-mapper-persistent-data ceph-common" && \
+# installation of yum-plugin-priorities must precede installation of epel-release in order for any potential
+# mounted override repo files to take effect
+RUN yum install -y yum-plugin-priorities && \
+    rpm -V yum-plugin-priorities && \
+    yum clean all
+
+RUN if [ -f /etc/yum.repos.d/local_epel.repo ]; then INSTALL_PKGS="which git tar wget hostname sysvinit-tools util-linux bsdtar \
```

Comment:
I thought the priority plugin would let us install `epel-release` always? The back we're doing in CI should be transparent to this container manifest.